src/calcPrimeNumbers/PrimeNumbers.java

```diff
@@ -0,0 +1,36 @@
+package calcPrimeNumbers;
+
+import java.util.Scanner;
+
+public class PrimeNumbers {
+	String startApp() {
+		System.out.println("Write an integer: ");
+		return null;
+	}
+
+	int inputFromKeyboard() {
+		Scanner keyboard = new Scanner(System.in);
+		int myint = keyboard.nextInt();
+		return myint;
+	}
+
+	public void isPrimeOrNot(int myint) {
```

Comment:
I would name this just isPrime. It's a good convention to do it like this. It makes more sense, imagine you would read the code like sentence.
e.g. if (isPrime(number)) {
...
}

You would read: "if number is prime then" (instead of "if number is prime or not then" which doesn't make sense).